share/picongpu/examples/IncidentField/include/picongpu/param/incidentField.param

```diff
@@ -33,42 +33,64 @@ namespace picongpu
     {
         namespace incidentField
         {
+            // This base class is merely to keep all laser-related things together
             class FunctorBase
             {
             public:
                 // Constants for pulse generation
-                static constexpr float_64 TIME_PERIOD_SI = 20 * SI::DELTA_T_SI;
-                static constexpr float_64 WAVE_LENGTH_SI = 2.0 * PI * TIME_PERIOD_SI * SI::SPEED_OF_LIGHT_SI;
-                static constexpr float_64 UNITCONV_A0_to_Amplitude_SI = -2.0 * PI / WAVE_LENGTH_SI
-                    * SI::ELECTRON_MASS_SI * SI::SPEED_OF_LIGHT_SI * SI::SPEED_OF_LIGHT_SI / SI::ELECTRON_CHARGE_SI;
+                static constexpr float_64 TIME_PERIOD_SI = 0.038e-9; // 0.038 ns
+                static constexpr float_64 WAVE_LENGTH_SI = TIME_PERIOD_SI * SI::SPEED_OF_LIGHT_SI;
+                static constexpr float_64 UNITCONV_A0_to_Amplitude_SI = -1.0 / WAVE_LENGTH_SI * SI::ELECTRON_MASS_SI
+                    * SI::SPEED_OF_LIGHT_SI * SI::SPEED_OF_LIGHT_SI / SI::ELECTRON_CHARGE_SI;
                 static constexpr float_64 A0 = 8.0;
                 static constexpr float_64 AMPLITUDE_SI = A0 * UNITCONV_A0_to_Amplitude_SI;
-                static constexpr uint32_t DURATION_STEPS = 300;
+                static constexpr float_64 DURATION_SI = 0.6e-9; // 0.6 ns

```

Comment:
For analysis of the laser pulse it would be helpful, when the full laser fits into the domain. In this case the duration should fulfill 16 * DURATION_SI * SPEED_OF_LIGHT_SI == 128 * dx, resulting DURATION_SI =  0.026e-9.
However, this is smaller than your TIME_PERIOD_SI.
In order to have a significant number of wavelengths in the laser, I would propose to change the time period to TIME_PERIOD_SI = 0.0211e-9, which results in a sampling of 11 time steps per laser period and 6 cells per wavelength.

Reply:
Did you mean `TIME_PERIOD_SI = 0.00211e-9` ? So that it's 11 times smaler than `DURATION_SI`, and in what you wrote it's roughly same.

Reply:
No. I mean it in the sense of TIME_PERIOD_SI = 11 * DELTA_T_SI. It is right that DURATIONS_SI ist then close to TIME_PERIOD_SI, but the laser will extend over several DURATION_SI, such that there are a few full wavelength in the laser field.